src/message/messagesActions.js

```diff
@@ -17,11 +17,13 @@ export const switchNarrow = (narrow: Narrow) => ({
   narrow,
 });
 
-export const doNarrow = (newNarrow: Narrow, anchor: number = Number.MAX_SAFE_INTEGER) =>
-  (dispatch: Dispatch, getState: GetState) => {
-    registerAppActivity(getAuth(getState()));
-    requestIdleCallback(() => dispatch(switchNarrow(newNarrow)));
-  };
+export const doNarrow = (newNarrow: Narrow, anchor: number = Number.MAX_SAFE_INTEGER) => (
+  dispatch: Dispatch,
```

Comment:
I don't think this formatting is an improvement. It is inconsistent.
A function definition on one line, then another one on another line seems quite logical (as it was originally).